index.html

```diff
@@ -242,7 +242,7 @@ <h3><span><img src="images/Rectangle 57.png" alt=""></span>&nbsp; Skills</h3>
                 </span>
                 <input  type="email" placeholder="Email address">
                 <textarea name="comments" id="" cols="30" rows="3" placeholder="Write me something..."></textarea>
-                <button>Get in touch</button>
+                <button class="contact-btn" type="button">Get in touch</button>
             </form>
         </div>
 
```

Comment:
- [ ] You did great using a `ul` and li items to lay out the social links on the website footer, however, following the best practices, I'll suggest you fill in the correct alt text on the image tags, it helps screen readers identify each image correctly :+1: :smile: 
![Screenshot from 2023-06-13 13-41-11](https://github.com/iankiema/My-Portifolio-Personal/assets/107632780/d9c865b8-3143-44fc-96e3-82743e9ee81c)

- -